app/views/layouts/application.html.erb

```diff
@@ -13,6 +13,10 @@
   <body>
     <%= render template: 'layouts/header' %>
     <%= yield %>
-    <%= render template: 'layouts/footer' %>
+<!--    TODO: background for homeController#index should cover entire page including footer-->
```

Comment:
I'm not a big fan of creating tech debt for a footer work that is fairly small

Reply:
@uuunk can chime in on this, so i'm flexible on the outcome of this

Reply:
@sanghun89 Didn't want to touch the login page css thats why I was doing this. I changed the way I was doing it now.